clustering/ee/cache/src/main/java/org/wildfly/clustering/ee/cache/scheduler/PortableConcurrentDirectDeque.java

```diff
@@ -1124,7 +1116,7 @@ public boolean addAll(Collection<? extends E> c) {
         // Copy c into a private chain of Nodes
         Node<E> beginningOfTheEnd = null, last = null;
         for (E e : c) {
-            checkNotNull(e);
+            checkNotNullParamWithNullPointerException("e", e);
```

Comment:
"e" is not a parameter of this method.  Is there an Assert.checkNotNullArrayParam(...) equivalent for collections?

Reply:
There is no equivalent method, but even when it would not be applicable to a non-parameter variable here. I have reverted the change.